### PR TITLE
chore: set dependabot for upgrading deps

### PR DESCRIPTION
**Pull request by kikoruiz**
## CHORE: set dependabot

**TASK**: #297

### Description, Motivation and Context
Set __GH Dependabot__ to upgrade our deps regarding we want to save the exact version of our dependencies.

### Types of changes

- [x] 🛠️ Tool

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    reviewers:
+      - '@adevinta/spark-dev'
+    ignore:
+      - dependency-name: '*'
+        update-types: ['version-update:semver-major']
```
Comment (kikoruiz) [inline .github/dependabot.yml:11]: For now, don't let it create PRs for major versions.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```
Comment (kikoruiz) [inline .github/dependabot.yml:6]: Just to do some tests right now. I think a good value could be `weekly`.

**Review by andresz1 — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -34,39 +34,39 @@
...
-    "@storybook/react": "^7.0.0-beta.43",
-    "@storybook/react-vite": "^7.0.0-beta.43",
-    "@storybook/testing-library": "^0.0.14-next.1",
-    "@testing-library/dom": "^8.20.0",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^14.4.3",
-    "@types/react": "^18.0.27",
-    "@types/react-dom": "^18.0.10",
+    "@storybook/addon-actions": "7.0.0-beta.43",
```
Comment (andresin87) [inline package.json:44]: all storybook might point to "next" branch, maybe beta, cuz right now we are in x.x.55

**Review by acd02 — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (kikoruiz) [reply package.json:44]: right!

**Pull request merged**